mkdocs.yml

```diff
@@ -164,6 +164,8 @@ nav:
           - docs/migrate/index.md
           - Migration strategy: docs/migrate/strategy.md
           - Migrate Quality Gates: docs/migrate/metrics-observe.md
+      - KeptnApp Migration:
```

Comment:
it's migration, maybe we can link it from upgrade...

Reply:
but this is not under a common migration element in the navigation 🤔 it will be a new floating "KeptnApp Migration" and it doesn't fit with the remaining top level elements

Reply:
I think the whole migration section strucutre should be changed. I would suggest something like this

Migration:
      - Migrating to Keptn:
          - docs/migrate/index.md
          - Migration strategy: docs/migrate/strategy.md
          - Migrate Quality Gates: docs/migrate/metrics-observe.md
      - KeptnApp Migration:


but didn't wanted to edit it in this PR. If we agree on adapting it here. I will do it. Or we can also make it in a follow-up

Reply:
adapted